index.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+let num = 0;
+
+
+const todos = [];
+const todo = {
+    comment: '',
+    status: '作業中'
+};
+
+const output = document.getElementById('output');
+
+//関数addTask
+const addTask = () => {
+    const input = document.getElementById('input').value;
+    todo.comment = input;
+    todos.push(todo);
+    displayTodos(todo);
+    num++
+};
+
+//関数displayTodos
+const displayTodos = (todo) => {
+    const list = document.createElement('td');
+    list.textContent = num;
+
+    const th = document.createElement('td');
+    th.textContent = todo.comment;
+    input.value = '';
+
+    const Status = () => {
+        const changeStatus = document.createElement('button');
+        changeStatus.textContent = todo.status;
+        return changeStatus;
+    };
+
+    const Delete = () => {
+        const changeDelete = document.createElement('button');
+        changeDelete.textContent = '削除';
+        return changeDelete;
+    };
+
+
+
+    const table = document.getElementById('new-task');
+    const tr = document.createElement('tr');
+    table.appendChild(tr);
+    table.appendChild(list);
+    table.appendChild(th);
+    table.appendChild(Status());
+    table.appendChild(Delete());
```

Comment:
西尾さんが提示してくださっているような HTML にするには以下のようなコードになっていないといけません

```suggestion
    table.appendChild(tr);
    tr.appendChild(list);
    tr.appendChild(th);
    tr.appendChild(Status());
    tr.appendChild(Delete());
```

`appendChild` はその英単語が示す通り「子要素を付け足す」ので
`<table>` の子要素である `<tr>` を追加するために
`table.appendChild(tr);` を実行する必要があり
`<tr>` の子要素である `<td>` を追加するために
`tr.appendChild(list);` を実行する必要があるんです。